src/Record.php

```diff
@@ -80,6 +80,15 @@ class Record extends PdoConnection
      */
     private $limit = array();
 
+    /**
+     * @var string The character set to use for table construction
+     */
+    private $character_set = 'utf8mb4';
+    /**
+     * @var string The collation to use for table construction
+     */
+    private $collation = 'utf8mb4_unicode_ci';
```

Comment:
worth noting that if these were left as `utf8` and `utf8_unicode_ci` it would be a backwards compatible update which would be preferable. Then you could cut a version 3 that updates the defaults. At least then, v2 users could choose to make the change voluntarily.

Reply:
Yea, backward compatibility is always, my preference but this is still getting loaded as a component in  a lot of places Blesta.  This means that every plugin file or other location that loads this would need to be updated to manually call the set functions to use mb4.

Since we are trying to shift Blesta entirely to that collation in v5, those kinds of manual calls are something I'd rather avoid.

Reply:
Lol missed the last part, yea I suppose I could add the manual update feature to v2 as well while I'm at it :)